fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -966,6 +967,7 @@ fun Drawer(
     behaviorType: BehaviorType = BehaviorType.BOTTOM,
     drawerState: DrawerState = rememberDrawerState(),
     scrimVisible: Boolean = true,
+    placeRelativeToAnchor: Boolean = false,
     offset: IntOffset = IntOffset(0, 0),
```

Comment:
Let make offset nullable. For null value pass clipingEnable = true in PopupPropeties & use offset (0,0) bydefault. If offset provided then pass clipingEnable = false & use the same offset. This way changes would be non-breaking at both Officemobile & Loop.